client/src/ui/deckbuilder/DeckBuilderExportMenu.vue

```diff
@@ -56,34 +68,48 @@ export default Vue.extend({
     },
 
     exportedDeck(): string {
-      return this.deck ? deckToXml(this.deck) : '';
+      return this.deck ? exportLoader.chunk.deckToXml(this.deck) : '';
```

Comment:
Don't you have to check libLoaded everywhere you do exportLoader.chunk?

Reply:
These computed values are only called when necessary and everything that requires them is wrapped in the <template> that checks if the lib is loaded, so we should be okay.